recipes/lz4/lz4_r131.oe

```diff
@@ -0,0 +1,5 @@
+require ${PN}.inc
+
+SRC_URI += "file://windows-h-include.patch"
+SRC_URI += "file://exeext.patch"
```

Comment:
Double (or more) negation sucks :)

Pro applying mingw patches conditionally: They are only relevant for mingw.

Pro applying mingw patches unconditionally: Developers not doing mingw build themselves will see patch conflicts for linux builds also.  Patch is closer to a form that can be accepted by upstream.